packages/salesforcedx-vscode-apex-replay-debugger/package.json

```diff
@@ -58,7 +58,7 @@
     "clean": "shx rm -rf node_modules && shx rm -rf out && shx rm -rf coverage && shx rm -rf .nyc_output",
     "postinstall": "node ./node_modules/vscode/bin/install",
     "test": "npm run test:vscode-integration",
-    "test:vscode-integration": "node ../../scripts/run-vscode-integration-tests-with-top-level-extensions",
+    "test:vscode-integration": "node ../../scripts/run-vscode-integration-tests",
```

Comment:
The tests on this package do not need the extensions being spinned up with vscode.